bindings/experimental/distrdf/python/DistRDF/Profiling/__init__.py

```diff
@@ -0,0 +1,68 @@
+#  @author Giulio Crognaletti
+#  @author Vincenzo Eduardo Padulano
+#  @author Enric Tejedor
+#  @date 2022-08
+
+################################################################################
+# Copyright (C) 1995-2022, Rene Brun and Fons Rademakers.                      #
+# All rights reserved.                                                         #
+#                                                                              #
+# For the licensing terms see $ROOTSYS/LICENSE.                                #
+# For the list of contributors see $ROOTSYS/README/CREDITS.                    #
+################################################################################
+
+# Inform cling to activate dumping of profiling information for jitted code.
+# It is crucial that this instruction is executed before importing ROOT in the distributed workers.
+from __future__ import annotations
+
+from os import environ
+environ["CLING_PROFILE"]="1"
```

Comment:
As said in a previous comment, this line is *always* executed, thus always changing the environment variable on the client side. This should be avoided